Library/Formula/ice.rb

```diff
@@ -32,7 +32,10 @@ def install
         --enable-cxx
       ]
 
-      args << "--enable-java" if build.with? "java"
+      if build.with? "java"
+        args << "--enable-java"
+        inreplace "dist/Makefile.in", "@JAVACFLAGS@",  "@JAVACFLAGS@ -source 1.7 -target 1.7"
```

Comment:
It's an internal bug submission system, there is no public access. Also, the version of db used by Ice comes from ZeroC, which already includes various patches. I don't see why this one line patch is such a big deal? Especially considering Oracle is no longer supported 5.3 and this version is specific just for ZeroC Ice.